calico/felix/config.py

```diff
@@ -248,6 +248,16 @@ def __init__(self, config_path):
                            "spoofing their source IP.  (For example, "
                            "unprivileged containers.)",
                            False, value_is_bool=True)
+        self.add_parameter("ClusterSize", "Total cluster host count",
```

Comment:
I think you should remove this config param, once we start reporting cluster size, I think we'll do it a different way